README.md

```diff
@@ -9,7 +9,7 @@ You can get nvOC running on your mining rig in some different ways:
 ### Noob-proof: Pre-built images
```

Comment:
Remove (or better, comment-out) this "noob-proof" section as this beta branch requires to follow the next "for insiders" install approach as explained below.